package.json

```diff
@@ -82,6 +82,7 @@
     "core-js": "^3.2.1",
     "dotenv": "^8.1.0",
     "es6-promise": "^4.2.8",
+    "eslint-import-resolver-webpack": "^0.11.1",
```

Comment:
Not sure what this dep is doing, seems to work fine without it? Regardless needs to be a dev dep :)